lib/app/pages/create_folder_page/create_folder_page.dart

```diff
@@ -27,12 +27,12 @@ class CreateFolderPage extends StatelessWidget {
                 ),
               ),
               Text(
-                context.tr.createFolderPage_sourceLanguageText,
```

Comment:
you can leave the name of the intl variables as they were, just change the context of the translation